utils/importJSONFile.js

```diff
@@ -11,16 +12,22 @@ import path from "path";
 async function importJSONFile(filePath, url) {
   if (!filePath) throw new Error("No file path provided");
 
+  const platform = os.platform();
   let finalPath = "";
 
   if (typeof url === "string" && url.length > 0)
-    finalPath = url.split("/").slice(0, -1).join("/").replace("file://", "");
+    finalPath = platform === 'win32' ?
+      url.split("/").slice(0, -1).join("/").replace(/^file:\/\/\/?/, '') :
+      url.split("/").slice(0, -1).join("/").replace("file://", "");
 
   finalPath = path.join(finalPath, filePath);
 
-  const data = await readFile(finalPath, "utf8");
-
-  return JSON.parse(data);
+  try {
+    const data = await readFile(finalPath, "utf8");
+    return JSON.parse(data);
+  } catch (err) {
+    throw new Error(`File not found: ${finalPath}`);
```

Comment:
There is the possibility that the file exists and was found by the readFile command but was not a valid JSON and JSON.parse() failed trying to parse it

Maybe this could help to return a better error:

![image](https://github.com/user-attachments/assets/135f458c-5c01-4fcf-9aac-b5f2ecaca15b)

Reply:
If it is a syntax error you could throw a syntax error too:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SyntaxError#creating_a_syntaxerror

```js
if (error instanceof SyntaxError) throw new SyntaxError("File is not a valid JSON.");

throw new Error("File not found");
```